### ISSUE TITLE
Padding columns

### ISSUE THREAD

**Issue by RubySword** [beto-rodriguez/LiveCharts2#495]
Good afternoon, how to cope with such a situation:
![screen](https://user-images.githubusercontent.com/93975635/170455474-4254eca4-2cac-4b5f-a04f-f2602a687a34.png)

**Comment by mzb986** [comment]
Incomplete display will appear when all values are 0

**Comment by RubySword** [comment]
Is this a bug? Will it be repaired soon?

**Comment by beto-rodriguez** [comment]
Hi @RubySword and thanks for the report, can you please elaborate more, I do not fully understand the issue, what do you mean with padding columns? I see that your plot is strange, but please let me know exactly what is wrong

**Comment by beto-rodriguez** [comment]
The bug is in the labels? do you mean that the axis labels overlap?

**Comment by RubySword** [comment]
Yes, because of this, the interface breaks down a bit. I tried changing padding and margin values, but it doesn't help

**Comment by beto-rodriguez** [comment]
Thanks for the update!

I was able to identify and fix the issue with the referenced commit.

This fix will be included in the next version of the library, just update your NuGet package when the next version is released and the issue will be gone.

**Comment by RubySword** [comment]
Thanks!
